🧪 **Casey** — CI Failing ❌

**`lint-and-test (api)` failed:** TypeScript errors in `credentials.ts`:

```
credentials.ts(63,92): error TS2339: Property 'credentialType' does not exist on type 'ExpiryQueryParams | undefined'.
credentials.ts(64,84): error TS2339: Property 'alertLevel' does not exist on type 'ExpiryQueryParams | undefined'.
```

The `Parameters<typeof expiryService.findExpiring>[0]` resolves to `ExpiryQueryParams | undefined`. Need to cast the type directly (e.g. `as CredentialType`) or narrow the type. Please fix CI.

--- a/api/src/routes/credentials.ts
+++ b/api/src/routes/credentials.ts
@@ -1,6 +1,6 @@
 import { Router, type Request, type Response, type NextFunction } from 'express';
 import { z } from 'zod';
-import { PrismaClient } from '@prisma/client';
+import { PrismaClient, type CredentialType } from '@prisma/client';
 import { PrismaCredentialRepository } from '../repositories/prisma/credential.js';
 import {
   CredentialService,
@@ -8,7 +8,7 @@ import {
   PersonnelNotFoundError,
   InvalidLBPLicenseError,
 } from '../services/credential.js';
-import { CredentialExpiryService } from '../services/credential-expiry.js';
+import { CredentialExpiryService, type AlertLevel } from '../services/credential-expiry.js';
 
 const prisma = new PrismaClient();
 const repository = new PrismaCredentialRepository(prisma);
@@ -60,8 +60,8 @@ credentialsRouter.get(
       const results = await expiryService.findExpiring({
         days,
         personnelId,
-        credentialType: credentialType as Parameters<typeof expiryService.findExpiring>[0]['credentialType'],
-        alertLevel: alertLevel as Parameters<typeof expiryService.findExpiring>[0]['alertLevel'],
+        credentialType: credentialType as CredentialType | undefined,
+        alertLevel: alertLevel as AlertLevel | undefined,
       });
 
       res.json({

